includes/Classifai/Admin/BulkActions.php

```diff
@@ -121,9 +122,15 @@ public function bulk_action_admin_notice() {
 	 * @return array
 	 */
 	public function register_row_action( $actions, $post ) {
+		$post_types = get_supported_post_types();
+
+		if ( ! in_array( $post->post_type, $post_types, true ) ) {
+			return $actions;
+		}
+
 		$actions['classify'] = sprintf(
 			'<a href="%s">%s</a>',
-			esc_url( wp_nonce_url( admin_url( 'edit.php?action=classify&ids=' . $post->ID ), 'bulk-posts' ) ),
+			esc_url( wp_nonce_url( admin_url( sprintf( 'edit.php?action=classify&ids=%d&post_type=%s', $post->ID, $post->post_type ) ), 'bulk-posts' ) ),
```

Comment:
Looks redundant, the hook `handle_bulk_actions-edit-$post_type` works regardless of `post_type` query argument presence.

Reply:
@cadic It's not redundant but it can be a bit tricky to realize how it hooked on a single classification request at first glance. 

In order to see reproduce the issue, please enable classify for "page" post type only, and click on the "Classify" row action for any given page.  You will see nothing happens without passing the correct `post_type`.

`wp-admin/edit.php` needs to know `$post_type` to execute the dynamic hook (`"handle_bulk_actions-edit-$post_type"`) accordingly.

I hope this makes sense.